src/Dto/Response/SimPackage/CurrentSimPackageDto.php

```diff
@@ -0,0 +1,174 @@
+<?php
+declare(strict_types=1);
+
+namespace eSIM\eSIMCoreClient\Dto\Response\SimPackage;
+/**
+ *
+ */
+class CurrentSimPackageDto
+{
+    /**
+     * @var float
+     */
+    private float $dataUsage;
+    /**
+     * @var string
+     */
+    private string $status;
+    /**
+     * @var string
+     */
+    private string $activatedDate;
+    /**
+     * @var string|null
+     */
+    private ?string $endDate;
+    /**
+     * @var string
+     */
+    private string $expiredDate;
+    /**
+     * @var SimDetailDto
+     */
+    private SimDetailDto $simDetail;
```

Comment:
$simDetail $packageDetail 
olabilir 
$simDetailDto $packageDetailDto olabilir benim için farketmez projedeki genel kullanım nasılsa öyle olsun ama daha da önemlisi  2'si aynı oolsun yeterli 👍

Reply:
Bunlarin donusleri simDetail ve packageDetail olarak goruntuleniyor. Ayni gorunuyor.

Reply:
biri     private SimDetailDto $simDetail;
biri     private PackageDetailDto $packageDetailDto;

    private SimDetailDto $simDetail;
    private PackageDetailDto $packageDetail;
    
    2'si aynı olsun demek istedim